src/SGFTree.cpp

```diff
@@ -181,6 +185,16 @@ void SGFTree::populate_states() {
         }
     }
 
+    // time
+    it = m_properties.find("TM");
+    if (it != end(m_properties)) {
+        std::string maintime = it->second;
```

Comment:
Can be auto.

Reply:
Ah I had just copied the style of the surrounding code. Will give it all a once over with auto.